src/converters/artifact-converter.ts

```diff
@@ -1,33 +1,8 @@
-import * as cx from 'aws-cdk-lib/cx-api';
-import { getAccountId, getPartition, getRegion } from '@pulumi/aws-native';
 import { AppComponent } from '../types';
 
 /**
  * ArtifactConverter
  */
 export abstract class ArtifactConverter {
     constructor(protected readonly app: AppComponent) {}
-
-    /**
-     * Takes a string and resolves any CDK environment placeholders (e.g. accountId, region, partition)
-     *
-     * @param s - The string that contains the placeholders to replace
-     * @returns The string with the placeholders fully resolved
-     */
-    protected resolvePlaceholders(s: string): Promise<string> {
```

Comment:
This was left over from before and it doesn't look like it is used anywhere anymore.